internal/services/containerapps/helpers/container_apps.go

```diff
@@ -2566,10 +2569,24 @@ func SecretsSchema() *pluginsdk.Schema {
 
 				"value": {
 					Type:        pluginsdk.TypeString,
-					Required:    true,
+					Optional:    true,
 					Sensitive:   true,
 					Description: "The value for this secret.",
 				},
+
+				"key_vault_url": {
+					Type:         pluginsdk.TypeString,
+					Optional:     true,
+					ValidateFunc: validation.IsURLWithHTTPS,
```

Comment:
can we validate that this is a Key Vault Nested Item, since we don't support Managed HSM Nested Items in the provider at this time?